scripts/tools/zap/update_cluster_revisions.py

```diff
@@ -51,6 +70,34 @@ def getTargets(cluster_id: int):
     return targets
 
 
+def get_outdated_clusters(data: object, xml_clusters: dict, args) -> list[ClusterInfo]:
```

Comment:
The contents of args could have been state of a VersionUpdateProcessor class.

Overall this script relies too much on passing data around that could be state of a class